Sources/Core/AST/AST.swift

```diff
@@ -50,8 +58,15 @@ public struct AST {
     set { storage.coreLibrary = newValue }
   }
 
+  /// Information about the compiler processing `self`.
+  public var compilerInfo: CompilerInfo {
```

Comment:
```suggestion
  public var compiler: CompilerInfo {
```
There's no need to repeat the type of the property verbatim.

Reply:
fixed in all places.